config/webpack.config.js

```diff
@@ -219,8 +219,13 @@ const pluginConfig = function(plugin) {
     );
   });
 
+  if (config.mode == 'production') {
+    var outputPath = path.join(pluginRoot, 'public', 'webpack', pluginName);
```

Comment:
@evgeni Why was this change made? should we add a .gitignore to all plugins for it?

Reply:
When we compile production assets (= during package building), we don't have write access to the Foreman folder, so we place the assets into the plugin directory (and then create a symlink when the package is installed).
Without this change, RPM builds of plugins fail, as they don't find the assets where they expect them to be.

This matches this old code:
https://github.com/theforeman/foreman/blob/066dd49a9ec121038bd93ac7d03f28ea0fc4b69c/config/webpack.config.js#L88C9-L92
introduced in https://github.com/theforeman/foreman/commit/b0ee9fd635191cf890fa5f63e26f7830c829f552

As this is a previously used path, and only used in production (where we don't have Git checkouts), I don't think this deserves any `.gitignore` additions.